treeherder/services/pulse/sources.py

```diff
@@ -29,16 +29,14 @@
 
 # Get Push ingestion source locations.
 # Specifies the Pulse exchanges Treeherder will ingest data from for Push data.
-push_sources = env.json(
+# Routing keys are specified after a period ("."), separated by colons (":").
+# Routing keys use a colon separator to avoid problems with defining multiple
+# push sources in an environment which are comma separated.
+push_sources = env.list(
     "PULSE_PUSH_SOURCES",
```

Comment:
The current value for `PULSE_PUSH_SOURCES` on production is:
```
 [{"exchange": "exchange/taskcluster-github/v1/push", "routing_keys": ["#"]},
  {"exchange": "exchange/taskcluster-github/v1/pull-request", "routing_keys": ["#"]},
  {"exchange": "exchange/hgpushes/v1", "routing_keys": ["#"]}]
```

We'll want to unset `PULSE_PUSH_SOURCES` in all environments **before** this is merged to master, otherwise the changes here will break ingestion. (This should be safe to do before merging, since the defaults match the value set at the moment)

Or to simplify both the deployment and code even more, we could just make this a fixed constant and not allow environment variable configuration at all. I don't see this list changing any time soon - and people can just edit the file locally prior to them opening a PR up and changing the defaults on `master`.

Thoughts?

Reply:
I added the configurability for the "oh but what if" scenario but you've reminded me that YAGNI is great :)

I'll hardcode this and update the docs/PR description and explain why we did this in the comments.